server/routes/incidents.test.js

```diff
@@ -110,13 +110,13 @@ describe('GET /:reportId/your-statement', () => {
       }))
 })
 
-describe('POST /:reportId/your-statement', () => {
-  it('should save ammendment', () =>
+describe('POST /:reportId/add-comment-to-statement', () => {
```

Comment:
think you need a test for the  `GET` add-comment-to-statement

Reply:
done